src/Sulu/Bundle/MediaBundle/Resources/js/views/MediaOverview/MediaOverview.js

```diff
@@ -33,21 +49,36 @@ class MediaOverview extends React.Component<ViewProps> {
         this.mediaPage.set(1);
 
         router.bind('collectionPage', this.collectionPage, 1);
+        router.bind('mediaPage', this.mediaPage, 1);
         router.bind('locale', this.locale);
         router.bind('id', this.collectionId);
 
         this.disposer = autorun(this.createCollectionStore);
+
         this.createCollectionDatagridStore();
         this.createMediaDatagridStore();
 
         router.bind('search', this.mediaDatagridStore.searchTerm);
+        router.bind('collectionLimit', this.collectionDatagridStore.limit, 10);
+        router.bind('mediaLimit', this.mediaDatagridStore.limit, 10);
+
+        this.collectionLimitDisposer = autorun(
+            () => userStore.setPersistentSetting(getLimitSettingKey(COLLECTIONS_RESOURCE_KEY),
+                this.collectionDatagridStore.limit.get())
```

Comment:
Also if the function arguments don't fit in a line, they should get their own:

```javascript
() => userStore.setPersistentSetting(
    getLimitSettingKey(),
    this.collectionDatagridStore.limit.get()
);
```

I also thought we have already linted that, can you check if there is a rule?